hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -43,7 +43,7 @@
   public static final String STORAGE_ID = "storageID";
   public static final String DATANODE_UUID = "datanodeUuid";
   public static final String CLUSTER_ID = "clusterID";
-  public static final String LAYOUTVERSION = "layOutVersion";
+  public static final String LAYOUTVERSION = "layoutVersion";
```

Comment:
Changing this may break backward compatibility since the value of this constant is written down to and read from the Version file. We can keep this as it is.